k8s/base/jitsi-meet-front-ingress.yml

```diff
@@ -0,0 +1,27 @@
+# Note that the domain specified in this file (placeholder.domain.example) is
+# just a placeholder with an arbitrary value. It is replaced by kustomize
+# during the build phase (see replacements in kustomization.yaml)
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: "jitsi-meet-front"
+  labels:
+    app: "jitsi-meet-front"
+  annotations:
+    cert-manager.io/issuer: "letsencrypt-jitsi-issuer"
+spec:
+  rules:
+  - host: "placeholder.domain.example"
```

Comment:
make it a variable?

Reply:
Do you have a specific suggestion on how to do this?

This placeholder value is not used: it is replaced by the content of the `XMPP_DOMAIN` variable with the replacement feature of kustomize, as described in the comment on top of this file.

Reply:
Yes that's the part I'm not confortable with as we can end-up with example settings in production right?